staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -82,12 +82,12 @@ type Event struct {
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
 	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestObject runtime.RawExtension `json:"requestObject,omitempty"`
+	RequestObject runtime.Unknown `json:"requestObject,omitempty"`
```

Comment:
@timstclair the event objects wouldn't convert between unversioned and versioned without this change. Looks like the unversioned types went from RawMessage to Unknown, but the versioned types weren't changed